spec/features/admin/user_admin_page_spec.rb

```diff
@@ -15,16 +15,17 @@
   end
 
   context "with session" do
+    let(:barbara) { User.find_by(email: "birdsofprey@education.gov.uk") }
```

Comment:
Probably easier to assign this user to a variable at the start and hang on to it, but not a big thing